Update static.conf - y0sh add fix ipv4@mash

Club-Mate-Cola opened the pull request.

Club-Mate-Cola committed:
Update static.conf - y0sh add fix ipv4@mash

tackin commented:
Den kann ich so nicht mergen. Das macht keinen Sinn. Dein Hostname ist nicht Teil unserer tdl und du schreibst Kram an Stellen wo er nicht hin gehört.

Club-Mate-Cola committed:
add fix ip @ y0sh

jetzt aber richtig :-)
name und mac angepasst

Club-Mate-Cola commented:
habs geändert sollte jetzt passen oder habe ich noch irgenwo gefailt ?

tackin commented:
Ja! Schau dir bitte die Files changed an.

Club-Mate-Cola committed:
Update static.conf - y0sh add fix ipv4@mash

ohhh sorry habe ich nicht gesehen... war wohl ein wenig müde heute morgen :-)
sollte jetzt passen ;)

tackin commented:
Na bestens. 😀
